### ISSUE TITLE
New version 0.9.3 is broken

### ISSUE THREAD

**Issue by TMA-Solutions** [mblode/vscode-twig-language-2#103]
After I updated to 0.9.3 the extension doesn't work anymore.

![twig](https://github.com/mblode/vscode-twig-language-2/assets/54527010/d9ffebb5-0bd6-4e2c-ab9b-9c7b3b6098d9)

I tried disable every other extensions to make sure nothing was causing conflict but it still didn't work. Tried downgrading to 0.9.2 and it worked fine.

**Comment by ImKubass** [comment]
Same, I had to downgrade. Formating works, snippets works. But Syntax higlighting is broken for me

**Comment by michtio** [comment]
@mblode - same issues, an with all other devs on my team as well. Syntax highlighting is broken since the update.

**Comment by fgdaniel** [comment]
Same issue for me, syntax highlight don't work

**Comment by fdelaneau** [comment]
I was going crazy figuring the problem was on my side. 
No syntax highlighting but here to, but the formatting doesn't work either. 

Once I reverted to 0.9.2 everything works again.

**Comment by bradlocking** [comment]
Same here, highlighting was working okay, but that may be because I already had the Twig syntax extension installed. Downgrading to `0.9.2` works for me too

**Comment by wuhhh** [comment]
Just adding my voice - latest is broken, `0.9.2` works

**Comment by MattWilcox** [comment]
Chiming in, syntax hilighting broken, it's allllllll grey now.

Which, honestly, really makes me appreciate a feature you don't think about at all normally.

**Comment by Guiguimps** [comment]
Thanks @TMA-Solutions  for this extension.
See you for the next working version ;)

**Comment by mayinx** [comment]
First of all: Thanks a lot for this extension and all the work you are putting into this! Twig Language 2 seems to be the only viable solution to work with twig-files in VS Code at all... 

### Issues

But: Due to the recent extension-update, our Team has multiple issues:

- Suddenly syntax highlighting is gone 
- And what's worse: Twig templates that had - f.i. - 200+ lines shrink into 20 lines when formatted - completely illegible (see snippets below)
- And to top that: In some cases, parts of the markup are even deleted - like a closing `endfor` - just vanished (EDIT: I can't reproduce this error - so we can't guarantee it was really caused by the extension - though we got no other explanation)

It would be super awesome, if the next version could fix this issues... 

### Details:

This markup:

```
{% extends 'form_div_layout.html.twig' %}
{% block form_row -%}
  {% set inputClasses = 'form__control' %}
  {% set labelClasses = 'form__label' %}
  <div class="form__group {% if required %}required{% endif %}{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
    {{- form_label(form, null, {'label_attr': {'class': labelClasses }}) -}}
    {{- form_widget(form, {'attr': {'class': inputClasses }}) -}}
    {{- form_help(form) -}}
    {{- form_errors(form) -}}
  </div>
{%- endblock form_row %}
{%- block form_errors -%}
  {%- if errors|length > 0 -%}
    {%- for error in errors -%}
      <div class="text-sm text-red-600 pl-2 pt-2 {% if form.parent is not empty %}pointing{% endif %} sylius-validation-error" {{ sylius_test_html_attribute('validation-error', error.cause.propertyPath) }} {% if form.parent is empty %}style="margin-bottom: 10px;"{% endif %}>
        {{ error.message }}
      </div>
    {%- endfor -%}
  {%- endif -%}
{%- endblock form_errors -%}
{% block checkbox_row -%}
  {% set inputClasses = 'switch-input' %}
  {% set labelClasses = 'pl-2 w-full block' %}
  <div class="mb-2{% if required %} required{% endif %}{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
    <div class="flex justify-start items-center">
      {{- form_widget(form, {'attr': {'class': inputClasses }}) -}}
      {{- form_label(form, null, {'label_attr': {'class': labelClasses }}) -}}
    </div>
    <div>{{- form_errors(form) -}}</div>
  </div>
{%- endblock checkbox_row %}
{% block radio_row -%}
  {% set inputClasses = 'rounded-lg w-full block' %}
  {% set labelClasses = 'text-sm font-bold w-full block mb-2' %}
  <div class="field{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
    <div class="ui radio checkbox">
      {{- form_label(form, null, {'label_attr': {'class': labelClasses }}) -}}
      {{- form_widget(form, {'attr': {'class': inputClasses }}) -}}
      {{- form_errors(form) -}}
    </div>
  </div>
{%- endblock radio_row %}
{%- block radio_widget -%}
  <input type="radio" class="radio-input" {{ block('widget_attributes') }}{% if value is defined %} value="{{ value }}"{% endif %}{% if checked %} checked="checked"{% endif %} />
{%- endblock radio_widget -%}
{% block money_widget -%}
  {% set inputClasses = 'rounded-lg w-full block' %}
  {% set labelClasses = 'text-sm font-bold w-full block mb-2' %}
  <div class="">
    <div class="">{{ currency|sylius_currency_symbol }}</div>
    {{- form_widget(form, {'attr': {'class': inputClasses }}) -}}
  </div>
{%- endblock money_widget %}
{% block choice_row -%}
  {% set inputClasses = 'form__control' %}
  {% set labelClasses = 'form__label' %}
  <div class="mb-4 {% if required %}required{% endif %}{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
    {{- form_label(form, null, {'label_attr': {'class': labelClasses }}) -}}
    {% set attr = attr|merge({'class': attr.class|default ~ inputClasses}) %}
    {{- form_widget(form, {'attr': attr}) -}}
    {{- form_errors(form) -}}
  </div>
{%- endblock choice_row %}
{%- block choice_widget_expanded -%}
  {% set labelClasses = 'pl-2 font-medium' %}
  <div {{ block('widget_container_attributes') }}>
    <div class="flex justify-between px-4 {% if required %}required{% endif %}">
      {%- for child in form %}
        <div>
          {{- form_widget(child, sylius_test_form_attribute('option')) -}}
          {{- form_label(child, null, {'label_attr': {'class': labelClasses}, translation_domain: choice_translation_domain}) -}}
        </div>
      {% endfor -%}
    </div>
  </div>
{%- endblock choice_widget_expanded -%}
{% block percent_widget -%}
  {% set inputClasses = 'rounded-lg w-full block' %}
  <div class="">
    {{- form_widget(form, {'attr': {'class': inputClasses }}) -}}
    <div class="">%</div>
  </div>
{%- endblock percent_widget %}
{% block collection_widget -%}
  {% from '@SyliusResource/Macros/notification.html.twig' import error %}
  {% import _self as self %}
  {% set attr = attr|merge({'class': attr.class|default ~ ' controls collection-widget'}) %}
  {% apply spaceless %}
  <div data-form-type="collection" {{ block('widget_container_attributes') }} {% if prototype is defined and allow_add %} data-prototype="{{ self.collection_item(prototype, allow_delete, button_delete_label, prototype.vars.name)|e }}" data-prototype-name="{{ prototype.vars.name }}" {%- endif -%}>
    {{ error(form.vars.errors) }}
    {% if prototypes|default is iterable %}
      {% for key, subPrototype in prototypes %}
        <input type="hidden"
               data-form-prototype="{{ key }}"
               value="{{ self.collection_item(subPrototype, allow_delete, button_delete_label, subPrototype.vars.name)|e }}"
               data-subprototype-name="{{ subPrototype.vars.name }}" />
      {% endfor %}
    {% endif %}
    <div data-form-collection="list">
      {% for child in form %}{{ self.collection_item(child, allow_delete, button_delete_label, loop.index0) }}{% endfor %}
    </div>
    {% if prototype is defined and allow_add %}
      <a href="#" class="ui labeled icon button" data-form-collection="add">
        <i class="plus square outline icon"></i>
        {{ button_add_label|trans }}
      </a>passw
    {% endif %}
    </div>
  </div>
{% endapply %}
{%- endblock collection_widget %}
{% macro collection_item(form, allow_delete, button_delete_label, index) %}
  {% apply spaceless %}
  <div data-form-collection="item" data-form-collection-index="{{ index }}">
    {{ form_widget(form) }}
    {% if allow_delete %}
      <a href="#"
         data-form-collection="delete"
         class="ui red labeled icon button"
         style="margin-bottom: 1em">
        <i class="trash icon"></i>
        {{ button_delete_label|trans }}
      </a>
    {% endif %}
  </div>
{% endapply %}
{% endmacro %}
{% block sylius_province_widget %}
  <div class="ui compact segment">
    <div class="inline fields">
      {{ form_row(form.code) }}
      {{ form_row(form.name) }}
      {{ form_row(form.abbreviation) }}
    </div>
  </div>
{% endblock %}
{% block sylius_zone_member_widget %}
  <div class="ui compact segment">
    <div class="inline fields">{{ form_row(form.code) }}</div>
  </div>
{% endblock %}
{% block sylius_promotion_rule_widget %}<div class="ui segment">{{ block('form_widget') }}</div>{% endblock %}
{% block sylius_promotion_action_widget %}<div class="ui segment">{{ block('form_widget') }}</div>{% endblock %}
{% block sylius_promotion_configuration_widget %}
  <div class="configuration">
    {% for child in form %}
      <hr />
      {{ form_row(child) }}
      <hr />
    {% endfor %}
  </div>
{% endblock %}
{% block sylius_product_option_value_widget %}
  <div class="ui segment">
    {{ form_row(form.code) }}
    <div class="five fields">
      {% for locale, translationForm in form.translations %}
        {{ form_row(translationForm.value, {'label': locale|sylius_locale_name}) }}
        {% if 0 == loop.index % 5 %}
        </div>
        <div class="five fields">
        {% endif %}
      {% endfor %}
    </div>
  </div>
{% endblock %}
{% block sylius_translations_row -%}
  <h4 class="ui dividing header">{{ form_label(form) }}</h4>
  <div class="ui grid">
    {% for translation in form %}
      <div class="four wide column">{{ form_label(translation, translation.vars.name|sylius_locale_name) }}</div>
      <div class="twelve wide column">{{ form_widget(translation) }}</div>
      {{- form_errors(form) -}}
    {% endfor %}
  </div>
{%- endblock sylius_translations_row %}
{% block sylius_resource_autocomplete_choice_row %}
  <div class="{% if required %}required{% endif %}field{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
    {{- form_label(form) -}}
    <div class="sylius-autocomplete ui fluid search selection dropdown {% if multiple %}multiple{% endif %}"
         data-url="{{ remote_url }}"    <label class="form__label{% if required %} required{% endif %}" for="{{ id }}">
      {{ 'sylius.form.customer.plain_password' | trans }}
    </label>
         data-choice-name="{{ choice_name }}"
         data-choice-value="{{ choice_value }}"
         data-criteria-type="{{ remote_criteria_type }}"
         data-criteria-name="{{ remote_criteria_name }}"
         data-load-edit-url="{{ load_edit_url }}">
      {{- form_widget(form, {'attr': {'class' : 'autocomplete' }}) -}}
      <i class="dropdown icon"></i>
      <div class="default text">
        {% if placeholder is defined %}{{ placeholder|trans }}{% endif %}
      </div>
      <div class="menu"></div>
    </div>
    {{- form_errors(form) -}}
  </div>
{% endblock %}
{% block sylius_product_variant_generation_widget %}
  <div class="ui segment">
    <div class="ui two column stackable grid">
      <div class="column">{{ form_row(form.optionValues, {'label': false}) }}</div>
      <div class="column">
        {{ form_row(form.code) }}
        {{ form_row(form.name) }}
      </div>
    </div>
    <h4 class="ui dividing header">{{ 'sylius.ui.pricing'|trans }}</h4>
    {{ form_row(form.channelPricings, {'label': false}) }}
  </div>
{% endblock %}
{% block sylius_image_widget %}passw
  {% apply spaceless %}
  {{ form_row(form.type) }}
  <label for="{{ form.file.vars.id }}" class="ui icon labeled button">
    <i class="cloud upload icon"></i> {{ 'sylius.ui.choose_file'|trans }}
  </label>
  {% if form.vars.value.path|default(null) is not null %}
    <img class="ui small bordered image"
         src="{{ form.vars.value.path|imagine_filter('sylius_small') }}"
         alt="{{ form.vars.value.type }}" />
  {% endif %}
  <div class="ui hidden element">{{ form_widget(form.file) }}</div>
  <div class="ui element">{{- form_errors(form.file) -}}</div>
{% endapply %}
{% endblock %}

{% block sylius_gender_widget %}
  <div class="sliding-radiolist" data-sliding-radiolist>
    {% set checked = true %}
    {%- for position,choice in choices %}
      {% set id = id ~ '_' ~ position %}
      <input type="radio" {{ block('widget_attributes') }}{% if choice.value is defined %} value="{{ choice.value }}"{% endif %}{% if checked %} checked="checked"{% endif %} />
      {% set checked = false %}
      <label for="{{ id }}">{{ choice.label|trans }}</label>
    {% endfor -%}
    <span class="sliding-radiolist__glider" data-sliding-radiolist-glider></span>
  </div>
{% endblock %}

{% block password_widget %}
  <div class="form__group{% if required %} required{% endif %}">
    <label class="form__label{% if required %} required{% endif %}" for="{{ id }}">
      {{ 'sylius.form.customer.plain_password' | trans }}
    </label>
    {{ form_widget(form, {'attr': {'class': 'form__control', 'data-password-check data-js-password-reveal':null}}) }}
    <p class="password-check__requirements" data-password-check-req>
      <span id="minLength">
        <i class="ofp-icon--check-1"></i>
        8 {{ 'shop.checkout.register_form.requirements.characters' | trans }}
      </span>
      <span id="containsNumber">
        <i class="ofp-icon--check-1"></i>
        {{ 'shop.checkout.register_form.requirements.numbers' | trans }}
      </span>
      <span id="upperAndLower">
        <i class="ofp-icon--check-1"></i>
        {{ 'shop.checkout.register_form.requirements.uppercase' | trans }}
      </span>
    </p>
  </div>
{% endblock %}

{% block _sylius_checkout_select_shipping_neutralShipping_widget %}
  <div class="choice-listitem choice-listitem--with-price choice-listitem--with-visible-input" {{ sylius_test_html_attribute('neutral-shipping') }}>
    {{ form_widget(form, {'attr': {'class': 'switch-input' }}) }}
    <label class="choice-listitem__body choice-listitem__body--transparent" for="sylius_checkout_select_shipping_neutralShipping">
      <div class="choice-listitem__text">
        <div class="choice-listitem__title" {{ sylius_test_html_attribute('shipping-method-label') }}>
          {{ 'shop.checkout.select_shipping.neutral_shipping'|trans }}
        </div>
        <div class="choice-listitem__description">
        {{ 'shop.checkout.select_shipping.neutral_shipping_description'|trans }}
        </div>
      </div>
      <div class="choice-listitem__price" {{ sylius_test_html_attribute('shipping-method-fee') }}>
        <p>
          {{ 'shop.checkout.select_shipping.neutral_shipping_free'|trans }}
        </p>
      </div>
    </label>
  </div>
{% endblock %}
```

becomes this when formatted with the current version of Twig language 2:


```
{% extends 'form_div_layout.html.twig' %}
{% block form_row -%}
{% set inputClasses = 'form__control' %}
{% set labelClasses = 'form__label' %}
<div class="form__group {% if required %}required{% endif %}{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">
  {{- form_label(form, null, {'label_attr': {'class': labelClasses }}
  ) -}}
  {{- form_widget(form, {'attr': {'class': inputClasses }}
  ) -}}
  {{- form_help(form) -}}
  {{- form_errors(form) -}}
</div>{%- endblock form_row %}{%- block form_errors -%}{%- if errors|length > 0 -%}
{%- for error in errors -%}
  <div class="text-sm text-red-600 pl-2 pt-2 {% if form.parent is not empty %}pointing{% endif %} sylius-validation-error" {{ sylius_test_html_attribute('validation-error', error.cause.propertyPath) }} {% if form.parent is empty %} style="margin-bottom: 10px;" {% endif %}>
    {{ error.message }}
  </div>
{%- endfor -%}{%- endif -%}{%- endblock form_errors -%}{% block checkbox_row -%}{% set inputClasses = 'switch-input' %}{% set labelClasses = 'pl-2 w-full block' %}<div class="mb-2{% if required %} required{% endif %}{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}"><div class="flex justify-start items-center">
{{- form_widget(form, {'attr': {'class': inputClasses }}
) -}}
{{- form_label(form, null, {'label_attr': {'class': labelClasses }}
) -}}</div><div>
{{- form_errors(form) -}}</div></div>{%- endblock checkbox_row %}{% block radio_row -%}{% set inputClasses = 'rounded-lg w-full block' %}{% set labelClasses = 'text-sm font-bold w-full block mb-2' %}<div class="field{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}"><div class="ui radio checkbox">{{- form_label(form, null, {'label_attr': {'class': labelClasses }}) -}}{{- form_widget(form, {'attr': {'class': inputClasses }}) -}}{{- form_errors(form) -}}</div></div>{%- endblock radio_row %}{%- block radio_widget -%}<input type="radio" class="radio-input" {{ block('widget_attributes') }} {% if value is defined %} value="{{ value }}" {% endif %} {% if checked %} checked="checked" {% endif %}/>{%- endblock radio_widget -%}{% block money_widget -%}{% set inputClasses = 'rounded-lg w-full block' %}{% set labelClasses = 'text-sm font-bold w-full block mb-2' %}<div class=""><div class="">{{ currency|sylius_currency_symbol }}</div>{{- form_widget(form, {'attr': {'class': inputClasses }}) -}}</div>{%- endblock money_widget %}{% block choice_row -%}{% set inputClasses = 'form__control' %}{% set labelClasses = 'form__label' %}<div class="mb-4 {% if required %}required{% endif %}{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}">{{- form_label(form, null, {'label_attr': {'class': labelClasses }}) -}}{% set attr = attr|merge({'class': attr.class|default ~ inputClasses}) %}{{- form_widget(form, {'attr': attr}) -}}{{- form_errors(form) -}}</div>{%- endblock choice_row %}{%- block choice_widget_expanded -%}{% set labelClasses = 'pl-2 font-medium' %}<div {{ block('widget_container_attributes') }}><div class="flex justify-between px-4 {% if required %}required{% endif %}">{%- for child in form %}<div>{{- form_widget(child, sylius_test_form_attribute('option')) -}}{{- form_label(child, null, {'label_attr': {'class': labelClasses}, translation_domain: choice_translation_domain}) -}}</div>{% endfor -%}</div></div>{%- endblock choice_widget_expanded -%}{% block percent_widget -%}{% set inputClasses = 'rounded-lg w-full block' %}<div class="">{{- form_widget(form, {'attr': {'class': inputClasses }}) -}}<div class="">%</div></div>{%- endblock percent_widget %}{% block collection_widget -%}{% from '@SyliusResource/Macros/notification.html.twig' import error %}{% import _self as self %}{% set attr = attr|merge({'class': attr.class|default ~ ' controls collection-widget'}) %}{% apply spaceless %}<div data-form-type="collection" {{ block('widget_container_attributes') }} {% if prototype is defined and allow_add %} data-prototype="{{ self.collection_item(prototype, allow_delete, button_delete_label, prototype.vars.name)|e }}" data-prototype-name="{{ prototype.vars.name }}" {%- endif -%}>{{ error(form.vars.errors) }}{% if prototypes|default is iterable %}{% for key, subPrototype in prototypes %}<input type="hidden" data-form-prototype="{{ key }}" value="{{ self.collection_item(subPrototype, allow_delete, button_delete_label, subPrototype.vars.name)|e }}" data-subprototype-name="{{ subPrototype.vars.name }}"/>{% endfor %}{% endif %}<div data-form-collection="list">{% for child in form %}{{ self.collection_item(child, allow_delete, button_delete_label, loop.index0) }}{% endfor %}</div>{% if prototype is defined and allow_add %}<a href="#" class="ui labeled icon button" data-form-collection="add"><i class="plus square outline icon"></i>{{ button_add_label|trans }}</a>passw{% endif %}</div> </div> {% endapply %}{%- endblock collection_widget %}{% macro collection_item(form, allow_delete, button_delete_label, index) %} {% apply spaceless %} <div data-form-collection="item" data-form-collection-index="{{ index }}">{{ form_widget(form) }}{% if allow_delete %}<a href="#" data-form-collection="delete" class="ui red labeled icon button" style="margin-bottom: 1em"><i class="trash icon"></i>{{ button_delete_label|trans }}</a>{% endif %} </div> {% endapply %}{% endmacro %}{% block sylius_province_widget %} <div class="ui compact segment"><div class="inline fields">{{ form_row(form.code) }}{{ form_row(form.name) }}{{ form_row(form.abbreviation) }}</div> </div>{% endblock %}{% block sylius_zone_member_widget %} <div class="ui compact segment"><div class="inline fields">{{ form_row(form.code) }}</div> </div>{% endblock %}{% block sylius_promotion_rule_widget %} <div class="ui segment">{{ block('form_widget') }} </div>{% endblock %}{% block sylius_promotion_action_widget %} <div class="ui segment">{{ block('form_widget') }} </div>{% endblock %}{% block sylius_promotion_configuration_widget %} <div class="configuration">{% for child in form %}<hr/>{{ form_row(child) }}<hr/>{% endfor %} </div>{% endblock %}{% block sylius_product_option_value_widget %} <div class="ui segment">{{ form_row(form.code) }}<div class="five fields">{% for locale, translationForm in form.translations %}{{ form_row(translationForm.value, {'label': locale|sylius_locale_name}) }}{% if 0 == loop.index % 5 %}</div><div class="five fields">{% endif %}{% endfor %}</div> </div>{% endblock %}{% block sylius_translations_row -%}<h4 class="ui dividing header"> {{ form_label(form) }}</h4><div class="ui grid"> {% for translation in form %}<div class="four wide column">{{ form_label(translation, translation.vars.name|sylius_locale_name) }}</div><div class="twelve wide column">{{ form_widget(translation) }}</div>{{- form_errors(form) -}} {% endfor %}</div>{%- endblock sylius_translations_row %}{% block sylius_resource_autocomplete_choice_row %}<div class="{% if required %}required{% endif %}field{% if (not compound or force_error|default(false)) and not valid %} error{% endif %}"> {{- form_label(form) -}} <div class="sylius-autocomplete ui fluid search selection dropdown {% if multiple %}multiple{% endif %}" data-url="{{ remote_url }}" <label class="form__label{% if required %} required{% endif %}" for="{{ id }}"> {{ 'sylius.form.customer.plain_password' | trans }} </label> data-choice-name="{{ choice_name }}" data-choice-value="{{ choice_value }}" data-criteria-type="{{ remote_criteria_type }}" data-criteria-name="{{ remote_criteria_name }}" data-load-edit-url="{{ load_edit_url }}">{{- form_widget(form, {'attr': {'class' : 'autocomplete' }}) -}}<i class="dropdown icon"></i><div class="default text">{% if placeholder is defined %}{{ placeholder|trans }}{% endif %}</div><div class="menu"></div> </div> {{- form_errors(form) -}}</div>{% endblock %}{% block sylius_product_variant_generation_widget %}<div class="ui segment"> <div class="ui two column stackable grid"><div class="column">{{ form_row(form.optionValues, {'label': false}) }}</div><div class="column">{{ form_row(form.code) }}{{ form_row(form.name) }}</div> </div> <h4 class="ui dividing header">{{ 'sylius.ui.pricing'|trans }} </h4> {{ form_row(form.channelPricings, {'label': false}) }}</div>{% endblock %}{% block sylius_image_widget %}passw{% apply spaceless %}{{ form_row(form.type) }}<label for="{{ form.file.vars.id }}" class="ui icon labeled button"> <i class="cloud upload icon"></i> {{ 'sylius.ui.choose_file'|trans }}</label>{% if form.vars.value.path|default(null) is not null %} <img class="ui small bordered image" src="{{ form.vars.value.path|imagine_filter('sylius_small') }}" alt="{{ form.vars.value.type }}"/>{% endif %}<div class="ui hidden element"> {{ form_widget(form.file) }}</div><div class="ui element"> {{- form_errors(form.file) -}}</div>{% endapply %}{% endblock %}{% block sylius_gender_widget %}<div class="sliding-radiolist" data-sliding-radiolist> {% set checked = true %} {%- for position,choice in choices %}{% set id = id ~ '_' ~ position %}<input type="radio" {{ block('widget_attributes') }} {% if choice.value is defined %} value="{{ choice.value }}" {% endif %} {% if checked %} checked="checked" {% endif %}/>{% set checked = false %}<label for="{{ id }}">{{ choice.label|trans }}</label> {% endfor -%} <span class="sliding-radiolist__glider" data-sliding-radiolist-glider></span></div>{% endblock %}{% block password_widget %}<div class="form__group{% if required %} required{% endif %}"> <label class="form__label{% if required %} required{% endif %}" for="{{ id }}">{{ 'sylius.form.customer.plain_password' | trans }} </label> {{ form_widget(form, {'attr': {'class': 'form__control', 'data-password-check data-js-password-reveal':null}} ) }} <p class="password-check__requirements" data-password-check-req><span id="minLength"><i class="ofp-icon--check-1"></i>8{{ 'shop.checkout.register_form.requirements.characters' | trans }}</span><span id="containsNumber"><i class="ofp-icon--check-1"></i>{{ 'shop.checkout.register_form.requirements.numbers' | trans }}</span><span id="upperAndLower"><i class="ofp-icon--check-1"></i>{{ 'shop.checkout.register_form.requirements.uppercase' | trans }}</span> </p></div>{% endblock %}{% block _sylius_checkout_select_shipping_neutralShipping_widget %}<div class="choice-listitem choice-listitem--with-price choice-listitem--with-visible-input" {{ sylius_test_html_attribute('neutral-shipping') }}> {{ form_widget(form, {'attr': {'class': 'switch-input' }} ) }} <label class="choice-listitem__body choice-listitem__body--transparent" for="sylius_checkout_select_shipping_neutralShipping"><div class="choice-listitem__text"><div class="choice-listitem__title" {{ sylius_test_html_attribute('shipping-method-label') }}>{{ 'shop.checkout.select_shipping.neutral_shipping'|trans }}</div><div class="choice-listitem__description">{{ 'shop.checkout.select_shipping.neutral_shipping_description'|trans }}</div></div><div class="choice-listitem__price" {{ sylius_test_html_attribute('shipping-method-fee') }}><p>{{ 'shop.checkout.select_shipping.neutral_shipping_free'|trans }}</p></div> </label></div>{% endblock %}
```

**Comment by mblode** [comment]
My sincerest apologies! I've reverted the latest changes. This should be fixed in 0.9.4

**Comment by mblode** [comment]
0.12.0 replaces PrettyDiff and adds regression tests for the historical formatting failures. Please share any example that still breaks.
